tests/test_pump_manager.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from greenpithumb import pump_manager
+
+
+class PumpManagerTest(unittest.TestCase):
+
```

Comment:
This is pretty foreign/confusing to me and I'm having a tough time debugging and figuring out why my change is failing the "moisture_is_low" case as a result of that.